okio/src/test/kotlin/okio/ByteStringTest.kt

```diff
@@ -16,21 +16,67 @@
 
 package okio
 
+import okio.ByteString.Companion.decodeBase64
 import okio.ByteString.Companion.decodeHex
+import okio.ByteString.Companion.encodeUtf8
+import okio.internal.commonAsUtf8ToByteArray
 import kotlin.test.Test
 import kotlin.test.assertEquals
 import kotlin.test.assertFalse
+import kotlin.test.assertNotEquals
+import kotlin.test.assertSame
 import kotlin.test.assertTrue
+import kotlin.test.fail
+
+class CommonByteStringTest {
```

Comment:
neat!

Reply:
... I assume JUnit’s parameterized tests don’t work for us here because they’re not cross-platform? Too bad; they’re more capable than subclassing

Reply:
I couldn't find any information on parameterized unit tests for multiplatform. There is potential for a framework like [Spek](https://spekframework.org/), but from what I've seen it's not quite ready yet for multiplatform use.